src/components/pokemon/pokemon.mapper.ts

```diff
@@ -87,6 +87,12 @@ export class PokemonMapper {
             diskRadius: pkmStgs.camera.diskRadiusM,
             shoulderModeScale: pkmStgs.camera.shoulderModeScale
         };
+        // Pokemon Evolutions
+        pokemon.nextEvolutionBranches = (pkmStgs.evolutionBranch || []).map(branch => {
+            var ident = <EvolutionTree> Util.SnakeCase2Identifyable(branch.evolution)
```

Comment:
My fault I did not configure tslint; but can you only use let or const next? Don't use var type for the sake of conistency.